### PR TITLE
Update quarkus/operator-sdk dependencies

### PR DESCRIPTION
**Pull request by r00ta**
[Insert JIRA Issue number and update link](https://issues.redhat.com/browse/MGDOBR-) 

Please make sure that your PR meets the following requirements:

- [ ] Your code is properly formatted according to [this configuration](https://github.com/kiegroup/kogito-runtimes/tree/main/kogito-build/kogito-ide-config)
- [ ] Your commit messages are clear and reference the JIRA issue e.g: "[MGDOBR-1] - $clear_explanation_of_what_you_did"
- [ ] All new functionality is tested
- [ ] Pull Request title is properly formatted: `MGDOBR-XYZ Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains link to any dependent or related Pull Request
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket

### CODE REVIEW

File: shard-operator/src/test/java/com/redhat/service/bridge/shard/operator/ManagerSyncServiceTest.java [hunk 1]
```diff
@@ -51,7 +51,7 @@ public void testBridgesAreDeployed() throws JsonProcessingException, Interrupted
         String expectedJsonUpdateProvisioningRequest =
                 String.format("{\"id\": \"myId-1\", \"name\": \"myName-1\", \"endpoint\": \"myEndpoint\", \"customerId\": \"%s\", \"status\": \"PROVISIONING\"}", TestSupport.CUSTOMER_ID);
         String expectedJsonUpdateAvailableRequest =
-                String.format("{\"id\": \"myId-1\", \"name\": \"myName-1\", \"endpoint\": \"http://192.168.2.49/ob-myId-1\", \"customerId\": \"%s\", \"status\": \"AVAILABLE\"}",
+                String.format("{\"id\": \"myId-1\", \"name\": \"myName-1\", \"endpoint\": \"http://192.168.2.49/ob-myid-1\", \"customerId\": \"%s\", \"status\": \"AVAILABLE\"}",
```
Comment (danielezonca) [inline shard-operator/src/test/java/com/redhat/service/bridge/shard/operator/ManagerSyncServiceTest.java:54]: Is this related with the PR?

File: shard-operator/src/main/java/com/redhat/service/bridge/shard/operator/utils/LabelsBuilder.java [hunk 2]
```diff
@@ -111,8 +110,7 @@ public LabelsBuilder withComponent(String component) {
      */
     private String sanitizeAndCheckLabelValue(final String labelValue) {
         final String sanitized = KubernetesResourceUtil.sanitizeName(labelValue);
-        if (Utils.isNotNullOrEmpty(labelValue) &&
-                labelValue.length() <= KubernetesResourceUtil.KUBERNETES_DNS1123_LABEL_MAX_LENGTH) {
+        if (KubernetesResourceUtil.isValidName(sanitized)) {
```
Comment (danielezonca) [inline shard-operator/src/main/java/com/redhat/service/bridge/shard/operator/utils/LabelsBuilder.java:113]: Is `KubernetesResourceUtil.isValidName` replacing the previous check?

File: shard-operator/src/test/java/com/redhat/service/bridge/shard/operator/ManagerSyncServiceTest.java (hunk 1 above)
Comment (r00ta) [reply shard-operator/src/test/java/com/redhat/service/bridge/shard/operator/ManagerSyncServiceTest.java:54]: Yes the new `sanitizeName` provided by fabric8 has changed behaviour and makes the string lowercase

**Review by danielezonca — APPROVED** [review]

File: build-parent/pom.xml [hunk 3]
```diff
@@ -28,7 +28,7 @@
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     <quarkus.platform.artifact-id>quarkus-bom</quarkus.platform.artifact-id>
     <quarkus.platform.group-id>io.quarkus.platform</quarkus.platform.group-id>
-    <quarkus.platform.version>2.4.2.Final</quarkus.platform.version>
+    <quarkus.platform.version>2.6.1.Final</quarkus.platform.version>
```
Comment (robpblake) [inline build-parent/pom.xml:31]: @r00ta Shall we go to `2.6.2`

**Comment by robpblake** [comment]
@r00ta This upgrade should allow us to use the Amazon Services extension in Quarkus 2.6, which brings support for Secrets Manager. Should help us with #219

File: shard-operator/src/test/java/com/redhat/service/bridge/shard/operator/ManagerSyncServiceTest.java (hunk 1 above)
Comment (r00ta) [reply shard-operator/src/test/java/com/redhat/service/bridge/shard/operator/ManagerSyncServiceTest.java:54]: Just for the record.. 
Screenshot from 2022-01-17 08-54-45
while with the previous version of the library this test fails
